### DISCUSSION TITLE
Welcome to node-config Discussions!

### DISCUSSION THREAD

**Discussion by markstos** [node-config/node-config#696] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lorenwest** [comment]
Thank you for starting this.  It's a great place to discuss issues before adding them as a bug/feature/pull request in the repository.
